samples/comparison/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     compile "com.android.volley:volley:${VOLLEY_VERSION}"
     compile "com.nostra13.universalimageloader:universal-image-loader:${UIL_VERSION}"
     compile "com.squareup.picasso:picasso:${PICASSO_VERSION}"
+	compile 'com.jakewharton.picasso:picasso2-okhttp3-downloader:1.0.2'
```

Comment:
Can you please just fix this indentation? It should be four spaces rather than tabbed.

Otherwise, thanks for fixing this oversight.